### DISCUSSION TITLE
Unable to Slather due to missing .profraw/.profdata files

### DISCUSSION THREAD

**Discussion by jack-knife-1221** [lyndsey-ferguson/fastlane-plugin-test_center#371] [Q&A] [answered]
I had been using `scan` in Fastlane before and am planning to replace that with `mult_scan`. I am able to successfully run the `multi_scan`. I want to run slather after this step to generate Code coverage report but it Isgiving me an error saying it cannot find the profdata coverage directory. 

<img width="840" alt="image" src="https://user-images.githubusercontent.com/60683018/145777040-f13460fb-290a-413e-8519-b22a1e9194f7.png">

Upon comparison, I found that `scan` used to produce a _ProfileData_ folder at `project-directory/DerivedData/Build/ProfileData` and `multi_scan` is not doing so. 

I have tried generating the coverage report from xcresult but it does not give me coverage report with coverage percentage. 

Seeking help and guidance for the issue, also please let me know if I am missing anything.

<img width="535" alt="image" src="https://user-images.githubusercontent.com/60683018/145800608-4cb4c47c-6b78-461a-92ac-dec6fa736d5c.png">

**Comment by lyndsey-ferguson** [comment, marked as answer]
I want to apologize that I have not responded to this question. A family health issue is requiring me to narrow my focus on the essentials and I don't have the time I need to focus on the plugin more than a minute.  I suggest reviewing the issues as I feel that someone has already created an issue for this problem.
